app/src/main/java/cto/its/ny/gov/fakenews/util/constants/SourceCounntries.kt

```diff
@@ -0,0 +1,6 @@
+package cto.its.ny.gov.fakenews.util.constants
+
+enum class SourceCountries {
+    ae,ar,at,au,be,bg,br,ca,ch,cn,co,cu,cz,de,eg,fr,gb,gr,hk,hu,id,ie,il ,
+     it,jp,kr,lt,lv,ma,mx,my,ng,nl,no,nz,ph,pl,pt,ro,rs,ru,sa,se,sg,si,sk,th,tr,tw,ua,us,ve,za
+}
```

Comment:
A minor thing, but for `SourceCountries` and `SourceLanguages` you can make these upper case for consistency with enums and following the Kotlin coding conventions:

```suggestion
enum class SourceCountries {
    AE,AR,AT,AU,BE,BG,BR,CA,CH,CN,CO,CU,CZ,DE,EG,FR,GB,GR,HK,HU,ID,IE,IL ,
     IT,JP,KR,LT,LV,MA,MX,MY,NG,NL,NO,NZ,PH,PL,PT,RO,RS,RU,SA,SE,SG,SI,SK,TH,TR,TW,UA,US,VE,ZA
}
```

With that though, and in prep for the networking assignment (week 5), you'll need a way to map those values from their raw, lowercased strings to the enum:

```kotlin
enum class SourceCountries(val rawValue: String) {
    AE("ae"),
    AR("ar"),
    AT("at"),
    AU("au"),
    ...
}
```

This is more of a suggestion though at this point.